### ISSUE TITLE
How to convert camera paramaters to COLMAP format

### ISSUE THREAD

**Issue by farhadi76m** [colmap/colmap#2227]
Hello, 
I have a list metadata for my images, I wanna to reconstruct dense map. 
I have images and camera parameters which each images has a .txt file includes : 
   1. The intrinsic (K), 3x3 camera calibration matrix 
   2. Rotation matrix (3x3) from the world coordinate system to the camera local coordinate system
   3. Translation vector (3x1) from the world coordinate system to the camera local coordinate system
how can I convert them to COLMAP format ? could anyone show me how should I do ? 
I'm beginner , and have to make dense map. 
thank you

**Comment by tsattler** [comment]
Did you had a look at the tutorial here: https://colmap.github.io/faq.html#reconstruct-sparse-dense-model-from-known-camera-poses

**Comment by farhadi76m** [comment]
yes I look at it ! 
but I don't know what's my wrong ! 
![image](https://github.com/colmap/colmap/assets/47895920/9e98831c-7546-4240-9936-3a19be571695)
it's my camera coordination per image . then I built 3 .txt files. for images.txt , cameras.txt and points3d.txt (it's an empty file) . 
then I converted camera coordinators to a file that you mentioned in COLMAP docs. 
like this image : 
![image](https://github.com/colmap/colmap/assets/47895920/f5884008-b968-42ba-a3ec-a3db6c819b62)
but my major problem is how COLMAP use it? 
![image](https://github.com/colmap/colmap/assets/47895920/6f6b8289-63da-4f97-bde2-94ef6aaa9061)
nothing has changed in this section ! ! ! 
could you tell me what's my problem ? 
I'm confused . I've struggling with this problem for days I don't found any solution!

**Comment by tsattler** [comment]
The pose information stored in the database is extracted from images and their meta data.

Poses stored in 3D model files are not propagated into the database. Once you store the poses and intrinsics in the files, you can run the point_triangulator to obtain full Colmap models (including 3D points). You will need to extract and match features beforehand though.

**Comment by farhadi76m** [comment]
no!!! I mean I want to feed colmap with camera files! i have KRt files !

**Comment by tsattler** [comment]
I don't understand what you mean by "I want to feed colmap with camera files". If you want to store the orientations and positions in the database, the easiest way is probably to directly edit the database yourself. Note that the orientations and positions stored in the database are at most used for spatial matching, but not for the reconstruction process.

**Comment by farhadi76m** [comment]
Let me to explain it completely , for instance If we have 100 image from a drone , and KRt files for each images(grand truth), how can we use these files ? 
https://user-images.githubusercontent.com/47895920/277174158-9e98831c-7546-4240-9936-3a19be571695.png
each picture has its! I can not understand  it in COLMAP .

**Comment by ahojnnes** [comment]
Closing: how to use known camera poses with point_triangulator explained.
